rpm/generic/zfs.spec.in

```diff
@@ -376,7 +376,7 @@ systemctl --system daemon-reload >/dev/null || true
 /usr/lib/modules-load.d/*
 %{_unitdir}/*
 %{_presetdir}/*
-%{_generatordir}/*
+%{_systemdgeneratordir}/*
```

Comment:
We need a definition at the top of the spec for when it isn't defined...

For example: `%{!?_systemdgeneratordir: %global _systemdgeneratordir %{_prefix}/lib/systemd/system-generators}`